charts/edge/templates/statefulset.yaml

```diff
@@ -71,6 +84,12 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.labels['apps.kubernetes.io/pod-index']
+          {{- if .Values.enableK8sLogs }}
+          - name: EDGE_SELF_NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          {{- end }}
```

Comment:
Re: disk buffer. I was also thinking about adding control values for Buffer sizes:
https://github.com/answerbook/pipeline-service/blob/feature/edge-disk-buffer-2/lib/pipeline/base.js#L619-L631

We could include with PV changes, or just bring those in separately. Or if you don't want to deal with other nit picking I can bring PV in with those after the feature is formally in service `main`.

Reply:
I can add them in. Is it `MEZMO_DISK_BUFFER_MAX_BYTES` and `MEZMO_MEM_BUFFER_MAX_EVENTS` or are there others too?

Reply:
Thanks. Those are the only two coming so far.

Reply:
Added config options for those two